From 766d8272ab9d9ffea9f47c56df622c12eb33079f Mon Sep 17 00:00:00 2001
From: Huang Chaofan <chaofan.huang@nxp.com>
Date: Tue, 28 Aug 2018 16:33:03 +0800
Subject: [PATCH 4457/5242] MLK-19358 VPU: Enable VPU driver for 8qm mek B0

commit  3c769b5ca7428edde07a9a749391c9a34d75b8f9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable VPU driver for 8qm mek B0 by changing fsl-imx8qm-mek.dts

Signed-off-by: Huang Chaofan <chaofan.huang@nxp.com>
(cherry picked from commit 78ac6d510298c6add57d4d8d457c8064fe31bb04)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index 7b467cb..d6b4301 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -1354,3 +1354,15 @@
 		};
 	};
 };
+
+&vpu_decoder {
+	core_type = <2>;
+	status = "okay";
+};
+
+&vpu_encoder {
+	core_type = <2>;
+	status = "okay";
+};
+
+
-- 
1.7.9.5

